reference/5.1/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -433,13 +381,13 @@ the `Get-Service` cmdlet. It defines a member set named
 
 `<NoteProperty>`: Defines a property with a static value.
 
-The `<NoteProperty>` tag must have a pair of `<Name>` tags that specify
-the name of the new property and a pair of `<Value>` tags that specify
-the value of the property.
+The `<NoteProperty>` tag must have a pair of `<Name>` tags that specify the
```

Comment:
Should this be more explicit that the number of `<Name>` tags must equal the number of `<Value>` tags?  I'm not sure if saying "a pair of X" imparts that.

Reply:
fixed all of these